crates/hc_sandbox/src/save.rs

```diff
@@ -57,14 +57,16 @@ pub fn clean(mut hc_dir: PathBuf, sandboxes: Vec<usize>) -> anyhow::Result<()> {
             if entry.file_type()?.is_file() {
                 if let Some(s) = entry.file_name().to_str() {
                     if s.starts_with(".hc_live_") {
-                        std::fs::remove_file(entry.path())?;
+                        std::fs::remove_file(entry.path())
```

Comment:
These error when multiple tests start at the same time. It's not really important, if the file is gone then there's nothing to do. But it's unclear whether there's some actual problem. A bit of context helps make it clearer that this isn't an issue.